File: kotlin-staticmethod-example/src/test/java/app/HelloTest.kt
package app

import org.glassfish.jersey.server.ResourceConfig
import org.glassfish.jersey.test.JerseyTest
import org.junit.Test as test

import kotlin.test.expect

public class HelloTest : JerseyTest() {

  test fun testSay() {
    expect("Hello, world!") {
      target("hello")!!.queryParam("name", "world")!!.request()!!.get(javaClass<String>())!!
    }
  }

  override fun configure() = ResourceConfig().register(javaClass<Hello>())

}

File: hello-ava/hello-test.ts
import test from 'ava'
import Hello from './hello'

test.beforeEach(t => {
  t.context = new Hello()
})

test('hello default', t => {
  t.is(t.context.hello(), 'Hello, world!')
})

test('hello with arg', t => {
  t.is(t.context.hello('ava'), 'Hello, ava!')
})
